### DISCUSSION TITLE
不知多少年后，我终于打开了软件

### DISCUSSION THREAD

**Discussion by TotoWang-hhh** [DearVa/ExplorerEx#13] [Ideas]
**超级喜欢菜单的设计！**
不过该说的问题还是说下

兼容性问题：
1.很多电脑没有.Net 6.0，而且太新了，网上除了微软官网基本上没有
2.Win10虽然成功打开，不过很多按钮点了没反应（这个问题我会在issue再发一遍）

界面样式的建议：
1.这图标有点老啊![image](https://user-images.githubusercontent.com/67090055/159153498-5de3f00f-303a-4b74-8d2f-c3529819e784.png)
2.多用点亚克力效果（在Win7和Vista上全部换用毛玻璃，其他系统就算了），比如标题栏、界面背景、菜单等等（可以添加开关，让用户选择
3.动画可以不打开，但不能没有！！（意思是可以添加一个动画开关，个人不太在意软件运行速度

还有一些其他的建议：
菜单里的选项换成同步系统菜单里的选项，或者参考Win11的菜单，添加一个显示系统菜单的按钮
似乎没找到“新建”相关的菜单选项
在空白处按不出右键菜单
添加对网络盘的支持，详见下图：
![image](https://user-images.githubusercontent.com/67090055/159153612-1a03b5ce-b51a-4768-9da2-39d2db92f644.png)

**Comment by DearVa** [comment]
文件夹图标长这样，应该是渐变的问题，不太自然；
![Folder.svg](https://user-images.githubusercontent.com/55902546/159154033-c0f6c64d-0e29-490b-bb3d-1182e0fc1a16.png)
亚克力效果我应用到了所有的弹出窗口上（右键菜单、下拉按钮、添加书签的窗口等等），主界面我使用了Mica（云母）效果，因为我发现用亚克力太花了，看不清楚；
动画当然要有，这也是我开发这款软件的宗旨之一；
右键菜单以后会做成可自定义的，显示系统菜单也会有；
“新建”在左上角有个按钮，或者空白处右键（最新版本）；
网络磁盘、OneDrive已在路线图中，未来会开发

**Reply by TotoWang-hhh** [reply]
其实文件夹里换成一摞文件就够了，不然图标很小的时候看不清里面是啥，还以为是拟物风之类的

**Reply by TotoWang-hhh** [reply]
云母真心不好，参考一下这个界面，亚克力+半透明是个好东西
![image](https://user-images.githubusercontent.com/67090055/159156460-a082ff85-921c-434b-85dd-648bbe21fc00.png)

**Reply by TotoWang-hhh** [reply]
另外，我的Win10看不见亚克力效果！

**Reply by DearVa** [reply]
问题收到，因为目前调用的是win11的api，所以win10还显示不出来。
之前试过全亚克力效果，如图：
![8CC0F634-9051-4D9D-866C-C167DA2C17BB](https://user-images.githubusercontent.com/55902546/159156781-426e0ec1-7109-4c7f-9c75-f09450f1b5d1.png)
后期会支持主题切换

**Reply by TotoWang-hhh** [reply]
> 问题收到，因为目前调用的是win11的api，所以win10还显示不出来。 之前试过全亚克力效果，如图： ![8CC0F634-9051-4D9D-866C-C167DA2C17BB](https://user-images.githubusercontent.com/55902546/159156781-426e0ec1-7109-4c7f-9c75-f09450f1b5d1.png) 后期会支持主题切换

我觉得这样就挺好，之后在写设置这块的时候可以把这个选择权也交给用户，比如说模糊度设置、背景色透明度设置，包括模糊类型等等

**Reply by TotoWang-hhh** [reply]
> 问题收到，因为目前调用的是win11的api，所以win10还显示不出来。 之前试过全亚克力效果，如图： ![8CC0F634-9051-4D9D-866C-C167DA2C17BB](https://user-images.githubusercontent.com/55902546/159156781-426e0ec1-7109-4c7f-9c75-f09450f1b5d1.png) 后期会支持主题切换

问题是我现在这个Win11上还是没看见效果啊！！
详见#33，我在这里提到了这个问题。
